planex/cmd/clone.py

```diff
@@ -79,34 +87,27 @@ def main(argv=None):
 
     for pinpath in args.pins:
         pin = Link(pinpath)
-        reponame = basename(pin.url).rsplit(".git")[0]
-        checkoutdir = join(args.repos, reponame)
 
         if args.jenkins:
             print 'echo "Cloning %s"' % pin.url
-            clone_jenkins(pin.url, checkoutdir, pin.commitish, args.credentials)
+            clone_jenkins(pin.url, args.repos, pin.commitish, args.credentials)
```

Comment:
They inherit their names from the command line arguments.  We could override the variable names, but this way it is clear which argument sets the variable.